modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,32 +10,79 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate your plug-in by running the following command:
+. To deploy your plug-in to a cluster, install a Helm chart with the name of the plug-in as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option. Provide the location of the image within the `plugin.image` parameter by using the following command:
```

Comment:
I wonder if this would be easier to follow with a _where_ statement. Like:

```
To deploy your plug-in to a cluster, install a Helm chart with the name of the plug-in as the Helm release name by using the following command:

helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location:
+
[source,terminal]
----
helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
----
+
+
where:
+
--
`n <my-plugin-namespace>`:: Specifies an existing namespace to deploy your plug-in into.
<--create-namespace>:: Creates a new namespace to deploy your plug-in into.
`--set plugin.image=my-plugin-image-location`. Specifies the location of the image within the `plugin.image` parameter.
--

```

Reply:
I like this suggestion. Thanks!